Gemfile

```diff
@@ -7,6 +7,7 @@ ruby '3.0.4'
 gem 'rails', '~> 6.1.7'
 # Use sqlite3 as the database for Active Record
 gem 'sqlite3', '~> 1.4'
+gem 'devise'
```

Comment:
is devise in the database group? why not use the command to use Postgres as well... or change the Gem.

Reply:
Devise is not inside of any group at the time. I am going to configurate postgres in a moment.

Reply:
Understood. I will wait up for the next PR. 

Good job! 👍